projects/components/CHANGELOG.MD

```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add `actionsUpdate` emitter to `ActionMenuComponent`
 - Add `hideWhenEmpty` flag to `QuickSearchProvider` to control section display when there is no data
 
+### Fixed
+- BREAKING: Fixes the `//@ ts-ignore problem with the Widget Object finder`
```

Comment:
Can we add a bit of info saying that callers of `LocatorDriver.findWidget` must now specify the returned type as a generic since TS had trouble inferring it from its constructor in some cases